.github/workflows/lint-code.yml

```diff
@@ -0,0 +1,58 @@
+name: lint-code
+on: [push]
```

Comment:
I think per pull request would make more sense than for each push. Not sure what the syntax is though.